src/build/env.py

```diff
@@ -21,6 +20,11 @@
 from ._util import check_dependency
 
 
+Installer = typing.Literal['pip', 'uv']
```

Comment:
Do we wanna call this `uv-pip` instead of simply `uv`?

Reply:
I think the `--installer` implies uv pip. And if they added a “uv install”, we could transparently start using that instead.